make-release.sh

```diff
@@ -223,6 +239,21 @@ setupGitconfig() {
   export GITHUB_TOKEN="${CHE_BOT_GITHUB_TOKEN}"
 }
 
+while [[ "$#" -gt 0 ]]; do
+  case $1 in
+    '-t'|'--trigger-release') TRIGGER_RELEASE=1; NOCOMMIT=0; shift 0;;
+    '-v'|'--version') CHE_VERSION="$2"; shift 1;;
+    '-dv'|'--dwo-version') DWO_VERSION="$2"; shift 1;;
+    '-p'|'--phases') CHE_PARENT_VERSION="$2"; shift 1;;
+  esac
+  shift 1
+done
+
+if [[ ! ${CHE_VERSION} ]] || [[ ! ${DWO_VERSION} ]] || [[ ! ${PHASES} ]] ; then
+  usage
+  exit 1
```

Comment:
this exit should be part of the usage function, as we never show usage and then continue working. see comment above.